pkg/web/api/v1/user.go

```diff
@@ -27,9 +27,8 @@ type User struct {
 }
 
 type UserList struct {
-	Page         *PageQuery `json:"page"`
-	Users        []*User    `json:"users"`
-	AuthUserRole string     `json:"user_role"`
```

Comment:
What was `AuthUserRole`?

Reply:
I previously returned a string instead of a bool when I initially updated the UserList.